commands/permissions_test.go

```diff
@@ -54,3 +54,103 @@ func (s *MmctlUnitTestSuite) TestAddPermissionsCmd() {
 		s.Require().Equal(expectedError, err)
 	})
 }
+
+//---------------------------------------------------
+
+func (s *MmctlUnitTestSuite) TestRemovePermissionsCmd() {
+	s.Run("Removing a permission from an existing role", func() {
+		mockRole := &model.Role{
+			Id:          "mock-id",
+			Name:        "mock-role",
+			Permissions: []string{"view", "edit", "delete"},
+		}
+
+		expectedPatch := &model.RolePatch{
+			Permissions: &[]string{"view", "edit"},
+		}
+		s.client.
+			EXPECT().
+			GetRoleByName(mockRole.Name).
+			Return(mockRole, &model.Response{Error: nil}).
+			Times(1)
+		s.client.
+			EXPECT().
+			PatchRole(mockRole.Id, expectedPatch).
+			Return(&model.Role{}, &model.Response{Error: nil}).
+			Times(1)
+
+		args := []string{mockRole.Name, "delete"}
+		err := removePermissionsCmdF(s.client, &cobra.Command{}, args)
+		s.Require().Nil(err)
+	})
+
+	s.Run("Removing an existing permission from an existing role", func() {
```

Comment:
Maybe I'm missing something here but it looks like you're testing the same twice (this test seems to check for the same than the previous one) If that the case, I think this test should be removed.

I'd modify this test or a need one that checks for multiple permission removal in order to test that use case